app/views/admin/enterprises/_ng_form.html.haml

```diff
@@ -3,7 +3,6 @@
 -# So we use onchange and have to get the scope to access the ng controller
 = form_for [main_app, :admin, @enterprise], html: { name: "enterprise_form",
   "ng-controller" => 'enterpriseCtrl',
-  'onchange' => 'angular.element(enterprise_form).scope().setFormDirty()',
```

Comment:
I believe this one caused https://github.com/openfoodfoundation/openfoodnetwork/issues/2195.

Reply:
@Matt-Yorkley can you reopen a PR to fix that again then? I believe in v1.13 it' causing some issues in some instances so would be great to fix ASAP...

Reply:
Oh seems @mkllnk fixed that in https://github.com/openfoodfoundation/openfoodnetwork/pull/2196 that was merged today so it seems it is done.

Reply:
Oups nop sorry Maikel just did a PR to revert those changes so you still need to work on it @Matt-Yorkley :-(